contracts/token/TwistedSisterToken.sol

```diff
@@ -41,11 +43,16 @@ contract TwistedSisterToken is CustomERC721Full, ITwistedSisterTokenCreator {
         _;
     }
 
-    constructor (string memory _tokenBaseURI, ITwistedSisterAccessControls _accessControls, uint256 _transfersEnabledFrom)
-    public CustomERC721Full("Twisted", "TWIST") {
+    constructor (
+        string memory _tokenBaseURI,
+        ITwistedSisterAccessControls _accessControls,
+        uint256 _transfersEnabledFrom,
+        TwistedSisterArtistFundSplitter _artistFundSplitter
+    ) public CustomERC721Full("twistedsister.io", "TWIST") {
         accessControls = _accessControls;
         tokenBaseURI = _tokenBaseURI;
         transfersEnabledFrom = _transfersEnabledFrom;
+        artistFundSplitter = _artistFundSplitter;
```

Comment:
Add a mutator so we can switch out if needed

Reply:
yeah. good idea.